src/renderers/testing/ReactTestRendererFiberEntry.js

```diff
@@ -285,14 +285,19 @@ function toJSON(inst: Instance | TextInstance): ReactTestRendererNode {
   }
 }
 
-function nodeAndSiblingsArray(nodeWithSibling: ?Fiber) {
+function nodeAndSiblingsTrees(nodeWithSibling: ?Fiber) {
   var array = [];
   var node = nodeWithSibling;
   while (node != null) {
     array.push(node);
     node = node.sibling;
   }
-  return array;
+  const trees = array.map(toTree);
+  return trees.length ? trees : null;
+}
+
+function hasSiblings(node: ?Fiber) {
+  return node && node.sibling;
```

Comment:
Similar thing here. Explicit null checks `node == null && node.sibling === null` are faster, so we favor those in the Fiber codebase. Not a blocker.

Reply:
Strict or loose checks (`==` or `===`)? Also, is this also true for an explicit inequality check (`!==`)?

Reply:
Oops yes, that was a typo, I mean `node === null`. And yeah, same thing.